action.yml

```diff
@@ -43,7 +47,10 @@ runs:
         env:
           TAILSCALE_AUTHKEY: ${{ inputs.authkey }}
           ADDITIONAL_ARGS: ${{ inputs.args }}
+          HOSTNAME: ${{ inputs.hostname }}
         run: |
           sudo tailscaled 2>~/tailscaled.log &
-          HOSTNAME="github-$(cat /etc/hostname)"
+          if [[ -z "${HOSTNAME}" ]]; then
```

Comment:
Double brackets are a bash-ism, and in the case shown here no bash-specific functionality is being used. Custom runners can run a Linux distribution with a shell which doesn't support bash-specific constructs, can you make this POSIX:
```
if [ -z "${HOSTNAME}" ]; then
```

Reply:
Makes sense! I've corrected it :)